test/func/fixtures/jquery.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fixture</title>
+  </head>
+  <body>
+    <div class="e2e-content"></div>
+    <div class="e2e-error"></div>
+    <script src="error-handler.js"></script>
+    <script src="../../../lib/little-loader.js"></script>
+    <script src="jquery/jquery.js"></script>
```

Comment:
OK with me.

(But for one last suggestion: in this case I think the key part is that they're loaded in parallel, maybe `parallel-jquery`? The important aspect of the test isn't really that it's jQuery we're loading, it's that there's contention for the global `jQuery` object because a bunch are loaded at once – kind of like the `_LLOAD_TEST` global in other tests, except there's on contention there because that particular test loads serially.)